docs/rfcs/rfc-035-authorization.md

```diff
@@ -0,0 +1,759 @@
+# Authorization
+
+This RFC documents the proposal to add new authorization functionality into the Neo4j GraphQL Library to replace the existing `@auth` directive.
+
+## Instantiation
+
+The following is a type proposal for auth configuration:
+
+```ts
+type JWKS = {
+  uri: string;
+}
+
+type AuthorizationConfig = {
+  secret: string | JWKS
```

Comment:
Would this not lose functionality as the current type for secret looks like this for JWKS:

```ts
jwksEndpoint: string | ((req: RequestLike) => string); // Probably the equivalent of the `uri` here
```

And it looks like this for jwts:

```ts
secret: jsonwebtoken.Secret | ((req: RequestLike) => jsonwebtoken.Secret);
```

Reply:
Indeed, I've updated this section, thanks!